### PR TITLE
Run linter with prettier 2

### PR DESCRIPTION
**Pull request by alubbe**
## Summary

Some files haven't had the linter applied to, so let's fix that :)

## Test plan

No new tests are needed

### CODE REVIEW

**Review by Siemienik — CHANGES_REQUESTED** [review]
This pr again introduce lots of changes, which has no sense. Lets we keep current style rules

**Comment by alubbe** [comment]
Agreed - I've reset the arrow-parens behaviour back to prettier v1 ('as-needed'). The other changes were always 'wrong', I think and they looks safe enough to merge

File: lib/utils/under-dash.js [hunk 1]
```diff
@@ -84,12 +84,7 @@ const _ = {
   },
 
   escapeHtml(html) {
-    return html
```
Comment (Siemienik) [inline lib/utils/under-dash.js:87]: I might it is a regression

File: lib/csv/csv.js [hunk 2]
```diff
@@ -2,9 +2,7 @@ const fs = require('fs');
 const fastCsv = require('fast-csv');
 const customParseFormat = require('dayjs/plugin/customParseFormat');
 const utc = require('dayjs/plugin/utc');
-const dayjs = require('dayjs')
-  .extend(customParseFormat)
```
Comment (Siemienik) [inline lib/csv/csv.js:6]: for me it a one step backward

File: lib/xlsx/xform/sheet/cf-ext/conditional-formattings-ext-xform.js [hunk 3]
```diff
@@ -1,50 +1,50 @@
...
-  }
-
-  onParserClose(name, parser) {
-    // model is array of conditional formatting objects
-    this.model.push(parser.model);
-  }
-}
-
-module.exports = ConditionalFormattingsExtXform;
+const CompositeXform = require('../../composite-xform');
```
Comment (Siemienik) [inline lib/xlsx/xform/sheet/cf-ext/conditional-formattings-ext-xform.js:1]: what was done here?

File: lib/xlsx/xform/sheet/cf/conditional-formattings-xform.js [hunk 4]
```diff
@@ -1,89 +1,89 @@
...
-          rule.style = options.styles.getDxfStyle(rule.dxfId);
-          delete rule.dxfId;
-        }
-      });
-    });
-  }
-}
-
-module.exports = ConditionalFormattingsXform;
+const BaseXform = require('../../base-xform');
```
Comment (Siemienik) [inline lib/xlsx/xform/sheet/cf/conditional-formattings-xform.js:1]: what was done here?

File: spec/integration/issues/issue-1328-xlsx-worksheet-reader-date.spec.js [hunk 5]
```diff
@@ -1,20 +1,33 @@
...
-  beforeEach(() =>
-    new Promise((resolve, reject) => {
-      const workbookReader = new ExcelJS.stream.xlsx.WorkbookReader(fs.createReadStream('./spec/integration/data/dateIssue.xlsx'), { worksheets: 'emit', styles: 'cache', sharedStrings: 'cache', hyperlinks: 'ignore', entries: 'ignore' });
-      workbookReader.read();
-      workbookReader.on('worksheet', worksheet => worksheet.on('row', row => rows.push(row.values[1])));
-      workbookReader.on('end', resolve);
-      workbookReader.on('error', reject);
-
-    })
+  beforeEach(
```
Comment (Siemienik) [inline spec/integration/issues/issue-1328-xlsx-worksheet-reader-date.spec.js:6]: could you ignore this rule? imo, before was better

File: spec/unit/xlsx/xform/compy-xform.js [hunk 6]
```diff
@@ -1,77 +1,77 @@
...
-
-  reconcile(model, options) {
-    this.children.forEach(child => {
-      child.xform.prepare(model[child.tag], options);
-    });
-  }
-}
-
-module.exports = CompyXform;
+const BaseXform = verquire('xlsx/xform/base-xform');
```
Comment (Siemienik) [inline spec/unit/xlsx/xform/compy-xform.js:1]: what was done here?

File: spec/unit/xlsx/xform/table/auto-filter-xform.spec.js [hunk 7]
```diff
@@ -1,42 +1,42 @@
...
-      return this.initialModel;
-    },
-    tests: ['prepare', 'render', 'renderIn', 'parse'],
-  },
-];
-
-describe('AutoFilterXform', () => {
-  testXformHelper(expectations);
-});
+const testXformHelper = require('../test-xform-helper');
```
Comment (Siemienik) [inline spec/unit/xlsx/xform/table/auto-filter-xform.spec.js:1]: what was done here?

File: spec/unit/xlsx/xform/table/filter-column-xform.spec.js [hunk 8]
```diff
@@ -1,38 +1,38 @@
...
-    },
-    tests: ['prepare', 'render', 'renderIn', 'parse'],
-    options: {index: 1},
-  },
-];
-
-describe('FilterColumnXform', () => {
-  testXformHelper(expectations);
-});
+const testXformHelper = require('../test-xform-helper');
```
Comment (Siemienik) [inline spec/unit/xlsx/xform/table/filter-column-xform.spec.js:1]: what was done here?

File: spec/unit/xlsx/xform/table/table-column-xform.spec.js [hunk 9]
```diff
@@ -1,30 +1,30 @@
...
-    xml: '<tableColumn id="1" name="Foo" totalsRowFunction="Baz" />',
-    parsedModel: {name: 'Foo', totalsRowFunction: 'Baz'},
-    tests: ['render', 'renderIn', 'parse'],
-  },
-];
-
-describe('TableColumnXform', () => {
-  testXformHelper(expectations);
-});
+const testXformHelper = require('../test-xform-helper');
```
Comment (Siemienik) [inline spec/unit/xlsx/xform/table/table-column-xform.spec.js:1]: what was done here?

File: spec/unit/xlsx/xform/table/table-style-info-xform.spec.js [hunk 10]
```diff
@@ -1,48 +1,48 @@
...
-      return this.preparedModel;
-    },
-    tests: ['render', 'renderIn', 'parse'],
-  },
-];
-
-describe('TableStyleInfoXform', () => {
-  testXformHelper(expectations);
-});
+const testXformHelper = require('../test-xform-helper');
```
Comment (Siemienik) [inline spec/unit/xlsx/xform/table/table-style-info-xform.spec.js:1]: what was done here?

File: spec/unit/xlsx/xform/table/table-xform.spec.js [hunk 11]
```diff
@@ -1,23 +1,23 @@
...
-    xml: fs.readFileSync(`${__dirname}/data/table.1.2.xml`).toString(),
-    parsedModel: require('./data/table.1.3'),
-    tests: ['render', 'renderIn', 'parse'],
-  },
-];
-
-describe('TableXform', () => {
-  testXformHelper(expectations);
-});
+const fs = require('fs');
```
Comment (Siemienik) [inline spec/unit/xlsx/xform/table/table-xform.spec.js:1]: an the last time, what was done here?

**Review by Siemienik — CHANGES_REQUESTED** [review]
I've added more comments, If is it fine to you I'm going to propose some rules changes according to my comments 😉

File: lib/xlsx/xform/sheet/cf/conditional-formattings-xform.js (hunk 4 above)
Comment (Siemienik) [reply lib/xlsx/xform/sheet/cf/conditional-formattings-xform.js:1]: break line style

**Review by Siemienik — APPROVED** [review]
After some investigation, I changed max-len option to 100. Before was 160 - that value is too much for properly priettier 2 working. 

As effect some improvements was introduced like:

https://github.com/exceljs/exceljs/pull/1477/files#diff-a083b92106d7d843e5f325cb350dbf25L102-R109

**Comment by alubbe** [comment]
Looks good, I also found another typo

**Pull request merged**